protocol/messenger.go

```diff
@@ -2601,6 +2601,11 @@ func (m *Messenger) SyncDevices(ctx context.Context, ensName, photoPath string,
 		return err
 	}
 
+	err = m.syncAccountsPositions(rawMessageHandler)
```

Comment:
I haven't see corresponding changes for [HandleSyncRawMessages](https://github.com/status-im/status-go/blob/f806a0b8cfde8cb4873edccf4f7ea0d0eb07615a/protocol/messenger_sync_raw_messages.go#L19) ?